sdks/python/scripts/generate_pydoc.sh

```diff
@@ -58,6 +58,7 @@ excluded_patterns=(
     # TODO(BEAM-4543): Remove datastore paths once they no longer exist.
     apache_beam/io/gcp/datastore/v1/
     apache_beam/io/gcp/datastore_write_it_*
+    apache_beam/io/gcp/bigquery_io_*
```

Comment:
I didn't know we should exclude IT's here. Can we exclude a more generic pattern, for example anything in `apache_beam/io/gcp/` containing `_it_` or ending with `_test.py`?

A better solution may be to move all test-related code into `test/` folders. This may need to be done as a broader cleanup is probably out of scope of this change.

Reply:
`*_test.py` pattern is already here and my new name matches it.